### DISCUSSION TITLE
/trust public in main claim overriding all subclaims?

### DISCUSSION THREAD

**Discussion by dangn511** [GriefPrevention/GriefPrevention#972] [Q&A] [answered]
I'm a player on a server with GriefPrevention plugin

What I'm doing: First I made a claim, which I stand inside and do `/trust public`. Next, I made a subdivision, which I stood inside and do `/untrust public`. I double checked with `/trustlist` while standing inside the subdivision, and there was no explicit permission for building for anyone.

What happened: Other players can still build inside the subdivision

What I expected: I don't want people to build inside a subdivision that is part of a claim with /trust public

**Comment by RoboMWM** [comment]
there's a command called /restrictedsubclaims or something that will make claims that don't inherit trust from parent claim.

**Comment by Jikoo** [comment, marked as answer]
Closing due to lack of response, assuming resolved. The exact command is `/RestrictSubclaim`.
